repository-data/webfiles/src/main/resources/site/freemarker/hstdefault/essentials-banner.ftl

```diff
@@ -5,34 +5,34 @@
 <#assign hasLink = document.link?? />
 <#assign hasImage = document.image?has_content />
 
-<div class="grid-wrapper grid-wrapper--article">
```

Comment:
Can I take it that you have checked that other places (in other channels) continue to look as expected with `grid-wrapper grid-wrapper--article` removed?